test/integration/features/profile.feature

```diff
@@ -89,3 +89,16 @@ Scenario: User should be able to switch between non existing profiles
       oc cli context could not changed for 'minishift'. Make sure the profile is in running state or restart if the problem persists.
       Profile 'minishift' set as active profile
       """
+Scenario: User should not be able to create profile with blank profile name
+    When executing "minishift profile set"
+     Then exitcode should equal "1"
+     Then stdout should contain
```

Comment:
Is this test passing locally? I think it should be `Then stderr should contain`

Reply:
Right, i see, i can't run integration tests locally takes a lot of time, and the system become unusable :( So pushed it.

Reply:
Is the unit test fine? Checks only the empty string condition, not testing for correct argument.

Reply:
You could always push with `[WIP]` as state, and check the CI results for the message (as CI is able to run the integration tests).